mbof/views.py

```diff
@@ -23,20 +25,20 @@ def logout(request):
 
 
 def log_action(request):
-    me = request.user.__str__()
+    current_time = timezone.now()
```

Comment:
Is this browser time or server time?  That could make  a difference as either the user or server could be in a non-local timezone.